packages/block-library/src/separator/theme.scss

```diff
@@ -7,7 +7,7 @@
 
 	// Default, thin style
 	&:not(.is-style-wide):not(.is-style-dots) {
-		max-width: 100px;
+		width: 100px;
```

Comment:
I really don't love this line. It makes `wide` and `full` alignments on this block meaningless. Not sure if we could remove it though.

Reply:
that's an opinionated style, basically the default separator is just a small one. This PR just fixes the separator behavior that broke recently.

Reply:
right, I just think this style is a really bad one because its so hard to work around. TT1 Blocks opts into opinionated styles, and uses separators in the header and footer - even though they are set to align wide, the separator is forced to 100px, which isn't what the design wants. We could override it with specific widths for alignwide and alignfull but that seems to defeat the purpose of putting the alignment rules in Gutenberg.

Reply:
I suggest you open a dedicated issue and discuss with other designers. For now, I'd like to focus on making the editor and frontend match.

Reply:
I would echo Riad, I think there's a great opportunity to vastly improve our opinionated styles, to make them fantastic and if necessary, even more opinionated.